### PR TITLE
Refresh the build system

### PR DESCRIPTION
**Pull request by tilgovi**
- Remove yarn in favor of npm5.
- Remove flow due to patent concerns.
- Remove rollup and systemjs in favor of webpack.
- Remove node-es-module-loader in favor of @std/esm.
- Upgrade all dependencies.
- Add the skeleton of an index.html.
- Add a super simple demo.

### CODE REVIEW

**Comment by tilgovi** [comment]
Also note that this removes babel. If we make any release artifacts that are targeted at older environments we will cross that bridge when we come to it.

File: README.md [hunk 1]
```diff
@@ -9,14 +9,13 @@
...
-
 ```sh
-$ yarn # installs top level dependencies
-$ yarn run bootstrap # manages dependencies referenced in `packages/`
-$ yarn run serve # serves the demo at http://localhost:8080/
+$ npm install
+$ npm start
 ```
 
+Continue reading at [http://localhost:8080].
```
Comment (Treora) [inline README.md]: Brackets actually seem to prevent this from being marked up.

File: README.md [hunk 2]
```diff
@@ -9,14 +9,13 @@
 
 ## Development
```
Comment (Treora) [inline README.md]: Perhaps specify required node/npm version?

**Comment by BigBlueHat** [comment]
Huge thanks for this work @tilgovi and for the review @Treora!

**Pull request merged**